.github/workflows/frontend_lint.yaml

```diff
@@ -0,0 +1,20 @@
+name: frontend_format
+
+on:
+  push:
```

Comment:
Linting should be on `[ push ]`, no need for the additional `pull_request` triggers